template-parts/layout/header/content-person.php

```diff
@@ -34,12 +34,24 @@
 				<h1 class="mt-4 mb-3 pt-sm-2<?php echo $heading_color_class; ?>">
 					<?php echo $title; ?>
 				</h1>
+				<div class="mb-4">
+				<?php if ( $subtitles ) :
+				?>
+					<span class="lead d-inline-block mb-2<?php echo $heading_color_class; ?>">
+					<?php
+					foreach( $subtitles as $key => $title ) :
+						$job_title = $title['job_title'];
 
-				<?php if ( $subtitle ) : ?>
-				<span class="lead d-block mb-4 <?php echo $heading_color_class; ?>">
-					<?php echo $subtitle; ?>
-				</span>
+						echo $job_title;
+
+						if ( $key !== array_key_last( $subtitles ) ) {
+							echo ",";
+						}
+					?>
+					<?php endforeach; ?>
+					</span>
 				<?php endif; ?>
```

Comment:
You can probably replace all of this with `echo implode( ', ', $subtitles );`.

Reply:
thanks! updated: https://github.com/UCF/Main-Site-Theme/pull/382/commits/225c080820caa4ae54dd8ae821a155723c6a6cde